spec/fixtures/fixtures.rb

```diff
@@ -274,3 +274,17 @@ def next_year!
     self.day = 1
   end
 end
+
+class SingletonClassExample
+  # This turned out to be required line here to make singleton classes
+  # work properly under all platforms. Not sure if it worth trying to
+  # do something with it.
+  include Contracts
```

Comment:
I was forced to use an inclusion here regardless of `Contracts` being included already in `Object` (or somewhere in hierarchy) or not. Because using this is the only portable way to link eigenclass with its owner class.

I don't really know if it worth to try to get rid of it. I would rather fail with descriptive error when somebody try to use contracts inside of singleton class without making this inclusion first in original class.